content/spin/rust-components.md

```diff
@@ -208,31 +208,37 @@ Hello, there!
 
 If allowed, Spin components can send outbound HTTP requests.
 Let's see an example of a component that makes a request to
-[an API that returns random dog facts](https://some-random-api.ml/facts/dog) and
+[an API that returns random animal facts](https://random-data-api.fermyon.app/animals/json) and
 inserts a custom header into the response before returning:
 
 <!-- @nocpy -->
 
 ```rust
+use anyhow::Result;
+use spin_sdk::{
+    http::{Request, Response},
+    http_component,
+};
+
+/// Send an HTTP request and return the response.
 #[http_component]
-fn hello_world(_req: Request) -> Result<Response> {
-    let mut res = spin_sdk::http::send(
+fn send_outbound(_req: Request) -> Result<Response> {
+    let mut res = spin_sdk::outbound_http::send_request(
         http::Request::builder()
             .method("GET")
-            .uri("https://some-random-api.ml/facts/dog")
+            .uri("https://random-data-api.fermyon.app/animals/json")
             .body(None)?,
     )?;
-
     res.headers_mut()
-        .insert(http::header::SERVER, "spin/0.1.0".try_into()?);
-
+        .insert("spin-component", "rust-outbound-http".try_into()?);
+    println!("{:?}", res);
```

Comment:
I'd drop this line.  If a reader wants to try it out, they'll see the response over HTTP; if they don't, it's an extra line to figure out if it's important or not.